pyproject.toml

```diff
@@ -70,9 +70,10 @@ test = [
 execution = [
     { include-group = "test" },
     # Required to run the llvm integration tests
-    "maturin >=1.4.0,<2",
+    # The `execute_llvm` package itself should be installed directly
+    # by running `uv run maturin develop -m execute_llvm/Cargo.toml`.
+    "maturin >=1.7.7,<2",
     "pip >=24",
-    "execute-llvm",
```

Comment:
^ Removing the unversioned dependency from here seems to prevent `uv` from using an old cached version even if we've ran `maturin develop`.

Ideally `uv` would trigger `maturin` on its own, but there seems to be missing some rebuild triggers :/